docs/install/installtoK8.md

```diff
@@ -0,0 +1,123 @@
+# Install to Kubernetes (experimental)
+
+To install and provision Percona Everest to Kubernetes:
+{.power-number}
+
+1. Create the `percona-everest` namespace:
+    ```sh
+    kubectl create namespace percona-everest
+    ```
+2. Create a Kubernetes secret with an auto-generated root key used for encrypting secrets:
+    ```sh
+    ENCODED_SECRETS_ROOT_KEY=$(openssl rand -base64 32 | tr -d '\n' | base64); cat <<EOF | envsubst | kubectl apply -n percona-everest -f -
+    apiVersion: v1
+    kind: Secret
+    metadata:
+        name: everest-secrets-root-key
+    data:
+        secrets-root-key: $ENCODED_SECRETS_ROOT_KEY
+    EOF
+    ```
+
+    ??? example "Expected output"
+    
+        ```{.text .no-copy}
+        secret/everest-secrets-root-key configured
+        ```
+
+3. Deploy Everest to Kubernetes:
+
+    ```sh
+    kubectl apply -f https://raw.githubusercontent.com/percona/percona-everest-backend/v0.4.0/deploy/quickstart-k8s.yaml -n percona-everest
+    ```
+
+4. (Optional) Verify that the services started correctly:
+    
+    ```sh
+    kubectl get pods -n percona-everest
+    ```        
+    ??? example "Expected output"
+        ```
+        NAME                                                   READY   STATUS    RESTARTS       AGE
+        percona-everest-0                                      2/2     Running   2 (10s ago)    10s
+        ```
+
+5. Retrieve the external IP address for the Everest service. This is the address used for provisioning the cluster, and from where you can then launch Everest at the end of the installation procedure. In this example, the external IP address used is the default 127.0.0.1:  
+   
+    ```sh 
+    kubectl get svc/everest -n percona-everest
+    ```
+    
+    ??? example "Expected output"
+        ```
+        NAME      TYPE           CLUSTER-IP      EXTERNAL-IP     PORT(S)          AGE
+        everest   LoadBalancer   10.43.172.194   127.0.0.1       8080:31611/TCP   10s
+        ```
```

Comment:
Hey @catalinaadam @rasika-chivate we changed the way we deploy everest for security reasons. Now we need to provide the following options to the users. Please review it and improve it as you feel appropriate.
```suggestion
5. By default, the Everest UI/API is not exposed with an external IP. To access the UI/API, choose one of the following techniques to expose Everest:

    * Service Type Load Balancer - Change the everest service type to LoadBalancer:
    
      ```sh
      kubectl patch svc/everest -n percona-everest -p '{"spec": {"type": "LoadBalancer"}}'
      ```

    Retrieve the external IP address for the Everest service. This is the address used for provisioning the cluster, and from where you can then launch Everest at the end of the installation procedure. In this example, the external IP address used is the default 127.0.0.1:  
   
    ```sh 
    kubectl get svc/everest -n percona-everest
    ```
    
    ??? example "Expected output"
        ```
        NAME      TYPE           CLUSTER-IP      EXTERNAL-IP     PORT(S)          AGE
        everest   LoadBalancer   10.43.172.194   127.0.0.1       8080:31611/TCP   10s
        ```

    * Port Forwarding - Kubectl port-forwarding can also be used to connect to everest without exposing the service. 
    
      ```sh
      kubectl port-forward svc/everest 8080:8080 -n percona-everest
      ```
```

Reply:
@catalinaadam with this change I believe you no longer need to add the warning with discussed in Slack with Daniil. Maybe we should add a note to the Service Type Load Balancer part mentioning that it will expose everest to outside the k8s cluster and anyone with access to the IP generated by the load balancer will have access to everest.